client/src/components/interview/QuestionDisplay.js

```diff
@@ -1,23 +1,46 @@
-import React from 'react';
-import { TypeAnimation } from 'react-type-animation';
+import React, { useEffect } from "react";
+import { TypeAnimation } from "react-type-animation";
 
-const QuestionDisplay = ({ question, currentQuestionIndex, skipAnimate }) => (
-  <div className="mb-3 bg-white border-gray-100 text-sm md:text-2xl px-2 rounded-lg text-slate-600" style={{ minHeight: '80px' }}>
-    {question && (
-      skipAnimate ? (
-        <span>{question}</span>
-      ) : (
-        <TypeAnimation
-          key={currentQuestionIndex}
-          sequence={[question]}
-          wrapper="span"
-          speed={60}
-          style={{ display: 'inline-block' }}
-          repeat={0}
-        />
-      )
-    )}
-  </div>
-);
+const QuestionDisplay = ({ question, currentQuestionIndex, skipAnimate }) => {
+  // Function to handle text-to-speech
+  const speakQuestion = (text) => {
+    const synth = window.speechSynthesis;
```

Comment:
What happens if speechSynthesis is not supported by browser? (I'd like to see a demo of the TTS provided by google chrome)

Reply:
Yaa I will show you after the meet